packages/vuetify/src/components/VNavigationDrawer/VNavigationDrawer.sass

```diff
@@ -11,67 +11,110 @@
   .v-divider
     border-color: map-get($material, 'dividers')
 
+// Block
 .v-navigation-drawer
-  +bootable()
-
-  display: block
+  -webkit-overflow-scrolling: touch
```

Comment:
We use autoprefixer, looks like we shouldn't use prefixes here.

Reply:
This is only supported as the above in browsers that it is supported in.